### PR TITLE
Rename all to be changelog-update

### PR DESCRIPTION
**Pull request by keenahn**
Based on #10 , so rebase after it is merged.

1. Also added optional output for assert-changelog-update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,69 +35,70 @@ npm install -D @invisible/release-note
...
 ```
-$ push-release-note
+$ push-changelog-update
 ```
 
 2. Optional: set a name for your slack bot and an icon emoji in your `package.json`
 
 ```JSON
-  "releaseNote": {
+  "ChangelogUpdate": {
```
Comment (rodrigo4244) [inline README.md]: Why are you using a capital letter `C`? Shouldn't it be `changelogUpdate` in camelCase exactly as `slackbotName` and `iconEmoji`?

Comment (rodrigo4244) [reply README.md]: You forgot to add that the consumer can choose a custom changelog file (`get('ChangelogUpdate.changelogFile')`)

**Comment by rodrigo4244** [comment]
After fixing and merging PRs #9 and #10 and updating the changelog information here, feel free to merge.

File: README.md (hunk 1 above)
Comment (keenahn) [reply README.md]: I'll fix in the next PR thanks

**Pull request merged**